SU-HSDO/suhumsci
# Lead font experimental features toggle

# READY FOR REVIEW

## Summary
Utilize Experimental Features toggle to enable/disable Lead Font animations.

## Need Review By (Date)
asap

## Urgency
medium

## Steps to Test
1. Add Lead Font to a page with Experimental Feature toggle disabled
2. Confirm Lead Font does not animate
3. Enable Feature flag and repeat.
4. Switch Theme & repeat steps 1-3

🔥 Ensure there are no regressions resolved by https://github.com/SU-HSDO/suhumsci/pull/1003

## PR Checklist
- [PR Checklist](https://gist.github.com/sherakama/0ba17601381e3adbe0cad566ad4d80a5)
- [Sparkbox PR Checklist](../docs/SparkboxPRChecklist.md)
